source/endpoints/jobs.rst

```diff
@@ -0,0 +1,153 @@
+Code                   Description
+====================== ===============================================================
+200                    Revspeech API Job
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                          "id": "111111",
+                          "status": "in_progress",
+                          "created_on": "2018-05-05T23:23:22.29Z"
+                        }          
+---------------------- ---------------------------------------------------------------
+401                    Request Unauthorized
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                          "title": "Authorization has been denied for this request"
+                        }    
+---------------------- ---------------------------------------------------------------
+404                    Job Not Found
+====================== ===============================================================
+
+
+``POST /jobs``
+*****************
+
+Submits a transcription job
+
+**CURL Examples**
+
+.. code:: javascript
+
+    curl -X POST "https://api.rev.ai/revspeech/v1beta/jobs" -H "Authorization: Bearer <api_key>" -H "Content-Type: application/json" -d "{\"media_url\":\"https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3\",\"metadata\":\"This is a sample submit jobs option\"}"
+
+.. code:: javascript
+
+  curl -X POST "https://api.rev.ai/revspeech/v1beta/jobs" -H "Authorization: Bearer <api_key>" -H "Content-Type: multipart/form-data" -F "media=@media_file.mp3" -F "options={"metadata":"This is a sample submit jobs option for multipart"}"
+
+**Request Body (required)**
+
+====================== ===============================================================
+Content-Type           Example
+====================== ===============================================================
+application/json       Revspeech API Options
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                            "media_url": "https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3",
+                            "metadata": "This is a sample submit jobs option",
+                            "callback_url": "https://www.example.com/callback"
+                        }     
+---------------------- ---------------------------------------------------------------
+multipart/form         ``Example Value``
+
+                       .. code:: javascript
+
+                         
+                         Key      Value
+                         -------  --------------------------
+                         media    any media file with audio
+                         options  options model (either as a string or json file)
```

Comment:
I would clarify that this is the same options model as you should an example of in the json case.

I think the "either as a string or json file" is a bit cconfusing as well. I am not sure what you mean. are you talking about how to specify it with Curl - if so, i would omit it.